isis/tests/cnetcheckFunctionalTests.cpp

```diff
@@ -0,0 +1,159 @@
+#include <iostream>
+#include <QTemporaryDir>
+
+#include "cnetcheck.h"
+
+#include "Cube.h"
+#include "CubeAttribute.h"
+#include "PixelType.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "PvlKeyword.h"
+#include "TestUtilities.h"
+#include "FileName.h"
+#include "ControlNet.h"
+#include "SerialNumber.h"
+#include "ControlMeasure.h"
+#include "ControlPoint.h"
+
+#include "Fixtures.h"
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/cnetcheck.xml").expanded();
+
+TEST_F(ThreeImageNetwork, FunctionalTestCnetcheckCamera) {
+  QTemporaryDir prefix; 
+  ASSERT_TRUE(prefix.isValid());
+
+  QVector<QString> args = {"fromlist="+cubeListTempPath.fileName(), "prefix="+prefix.path()+"/", "nocube=false", "lowcoverage=false"};
+  UserInterface options(APP_XML, args);
+  
+  QString cube1Serial = SerialNumber::Compose(*cube1->label());
+  QString cube2Serial = SerialNumber::Compose(*cube2->label());
+  
+  // Add measure guarenteed to fail computing lat/lon
+  ControlMeasure *m1 = new ControlMeasure();
+  m1->SetAprioriLine(481);
+  m1->SetAprioriSample(481);
+  m1->SetCamera(cube1->camera());
+  m1->SetCubeSerialNumber(cube1Serial);
+ 
+  ControlMeasure *m2 = new ControlMeasure();
+  m2->SetAprioriLine(999);
+  m2->SetAprioriSample(999);
+  m2->SetCamera(cube2->camera());
+  m2->SetCubeSerialNumber(cube2Serial);
+  
+  ControlPoint *newPoint = new ControlPoint(); 
+  newPoint->Add(m1);
+  newPoint->Add(m2);
+  
+  network->AddPoint(newPoint);
+  
+  Pvl log;
+  QString results = cnetcheck(*network, *cubeList, options, &log);
```

Comment:
Small thing, I don't think we use results anywhere in the test.

Reply:
Originally I was gonna test against it but it really doesn't have super valuable information worth testing. Probably remove storing the results.